assets/airports/mkjp.json

```diff
@@ -177,19 +208,184 @@
+            },
+            "body": [["ENEKA", "A90+"]],
+            "exitPoints": {
+                "ENEKA": ["ENEKA"]
+            },
+            "draw": [
+                ["GUDIL", "ENEKA*"]
+            ]
+        },
+        "GUDIL5": {
+            "icao": "GUDIL5",
+            "name": "Gudil Five",
+            "rwy": {
+                "MKJP12": ["MLY"],
+                "MKJP30": [["GUDIL", "A28+"]]
+            },
+            "body": [["GUDIL", "A28+"]],
+            "exitPoints": {
+                "LIBEX" : ["LIBEX"],
+                "KEMBO" : ["KEMBO"],
+                "KESPA" : ["KESPA"],
+                "BOSOM": ["BOSOM"]
+            },
+            "draw": [
+                ["GUDIL", "LIBEX*"],
+                ["GUDIL", "KEMBO*"],
+                ["GUDIL", "KESPA*"],
+                ["GUDIL", "BOSOM*"]
+            ]
+        },
+        "KEYNO2": {
+            "icao": "KEYNO2",
+            "name": "Keyno Two",
+            "rwy": {
+                "MKJP12": [""],
+                "MKJP30": [""]
+            },
+            "body": ["KEYNO"],
+            "exitPoints": {
+                "KEYNO": ["KEYNO"]
+            },
+            "draw": [
+                ["KEYNO*"]
+            ]
+        },
+        "LEXUV5": {
+            "icao": "LEXUV5",
+            "name": "Lexuv Five",
+            "rwy": {
+                "MKJP12": [["UKIMO", "A4+"], ["PEMPE", "S220"]],
+                "MKJP30": [""]
+            },
+            "body": ["LEXUV"],
+            "exitPoints": {
+                "KEMBO": ["KEMBO"],
+                "RADOK" : ["RADOK"]
+            },
+            "draw": [
+                ["UKIMO", "PEMPE", "LEXUV*"],
+                ["LEXUV", "KEMBO*"],
+                ["LEXUV", "RADOK*"]
+            ]
+        },
+        "MLY7": {
+            "icao": "MLY7",
+            "name": "Manley Seven",
+            "rwy": {
+                "MKJP12": ["MLY"],
+                "MKJP30": ["MLY"]
+            },
+            "body": ["MLY"],
+            "exitPoints": {
+                "LIBEX": ["LIBEX"],
+                "KEMBO": ["KEMBO"],
+                "KESPA": ["KESPA"],
+                "BOSOM": ["BOSOM"]
+            },
+            "draw": [
+                ["MLY", "LIBEX*"],
+                ["MLY", "KEMBO*"],
+                ["MLY", "KESPA*"],
+                ["MLY", "BOSOM*"]
+            ]
+        },
+        "OMPAL3": {
+            "icao": "OMPAL3",
+            "name": "Ompal Three",
+            "rwy": {
+                "MKJP12": [""],
+                "MKJP30": ["GUDIl"]
+            },
+            "body": [""],
+            "exitPoints": {
+                "OMPAL": ["OMPAL"]
+            },
+            "draw": [
+                ["GUDIL", "OMPAL*"]
+            ]
+        },
+        "RADOK7": {
+            "icao": "RADOK7",
+            "name": "Radok Seven",
+            "rwy": {
+                "MKJP12": [""],
+                "MKJP30": ["ELSER"]
```

Comment:
Ok so this one's interesting. Looks like they've decided to add two procedures into one for RWY30 depending on aircraft category. I would suggest we choose one. You know the airport more than I do, so if the traffic is mostly small prop aircraft and small business jets, I would implement the A/B SID. If most of the traffic is larger jet aircraft, I would implement the C/D SID.

For the A/B SID, aircraft must fly a 17DME arc around MLY to intercept MLY R356 radial outbound. This would be done with a series of invisible waypoints along the arc. Have a look at the SAME and SAEZ airport files which feature arcs a lot.

The C/D SID has a right turn heading 027 at MLY 12DME to intercept MLY R356. It would look like this:
`["GUDIL", "^_MLY300012", "_MLY356025", RADOK]`
Where _MLY356025 is the intersection between MLY R356 and R027 from _MLY300012, which I estimate is roughly at MLY 25DME, but I would encourage you to find the actual distance of this point. [This website](https://www.movable-type.co.uk/scripts/latlong.html) is useful for those calculations